inspirehep/modules/records/mappings/records/authors.json

```diff
@@ -113,6 +113,15 @@
                             "index": "not_analyzed"
                         }
                     }
+                },
+                "self":{
```

Comment:
In order to be able to run the query at https://github.com/Panos512/inspire-next/blob/orcid/inspirehep/modules/orcid/tasks.py#L104 refs need to be not_analyzed since `Terms` keyword needs not_analyzed fields to find exact matches.